app/code/core/Mage/Customer/Model/Address/Config.php

```diff
@@ -101,7 +101,7 @@ public function getFormats()
                 $type = new Varien_Object();
                 $htmlEscape = strtolower((string)$typeConfig->htmlEscape);
                 $htmlEscape = !($htmlEscape == 'false' || $htmlEscape == '0' || $htmlEscape == 'no'
-                    || !strlen((string)$typeConfig->htmlEscape));
+                    || !strlen($htmlEscape);
```

Comment:
PHPStan reports an error. In line 104 there is a `)` missing before `;`

Reply:
```suggestion
                    || !strlen($htmlEscape));
```